easybuild/easyconfigs/b/BCFtools/BCFtools-1.8-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'BCFtools'
+version = '1.8'
+
+homepage = 'http://www.htslib.org/'
+
+description = """
+ Samtools is a suite of programs for interacting with high-throughput
+ sequencing data.  BCFtools - Reading/writing BCF2/VCF/gVCF files and
+ calling/filtering/summarising SNP and short indel sequence variants
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```

Comment:
same question as in #6209 and #6207, should this be a full toolchain like `intel`?

Reply:
Move to GCC level